ratis-server/src/test/java/org/apache/ratis/InstallSnapshotNotificationTests.java

```diff
@@ -220,7 +220,7 @@ private void testAddNewFollowers(CLUSTER cluster, int numRequests) throws Except
     // delete the log segments from the leader
     LOG.info("Delete logs {}", logs);
     for (LogSegmentPath path : logs) {
-      FileUtils.deleteFully(path.getPath()); // the log may be already purged
+      FileUtils.deleteFully(path.getPath()); // the log may be already puged
```

Comment:
I guess you might have accidentally edited the comment.

Reply:
Thank you very much for reviewing the code! This issue was indeed an oversight on my part. I will fix it along with the flaky tests in the upcoming revisions.